common/src/main/cpp/extensions/openxr_fb_passthrough_extension_wrapper.cpp

```diff
@@ -210,15 +210,24 @@ bool OpenXRFbPassthroughExtensionWrapper::start_passthrough() {
 	return true;
 }
 
-uint64_t OpenXRFbPassthroughExtensionWrapper::_get_composition_layer() {
-	if (is_passthrough_enabled()) {
+int OpenXRFbPassthroughExtensionWrapper::_get_composition_layer_count() {
+	return is_passthrough_enabled() ? 1 : 0;
+}
+
+uint64_t OpenXRFbPassthroughExtensionWrapper::_get_composition_layer(int p_index) {
+	if (p_index == 0) {
 		composition_passthrough_layer.layerHandle = passthrough_layer;
 		return reinterpret_cast<uint64_t>(&composition_passthrough_layer);
 	} else {
 		return 0;
 	}
 }
 
+int OpenXRFbPassthroughExtensionWrapper::_get_composition_layer_order(int p_index) {
+	// Ensure the passthrough layer will be below the projection layer.
+	return -9999;
```

Comment:
nit: Does it make sense to use `in front of` / `behind` instead of `below` / `above` when describing the layer ordering?

Reply:
Hm, yeah, I guess "behind" is a little clearer! Updated in latest push